crypto/src/main/java/net/consensys/cava/crypto/mikuli/SystemParameters.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.cava.crypto.mikuli;
+
+import net.consensys.cava.crypto.mikuli.group.G1Point;
+
+import org.apache.milagro.amcl.BLS381.BIG;
+import org.apache.milagro.amcl.BLS381.ECP;
+import org.apache.milagro.amcl.BLS381.ROM;
+
+public class SystemParameters {
+  static public final G1Point g1Generator = new G1Point(ECP.generator());
+  static public final BIG curveOrder = new BIG(ROM.CURVE_Order);
```

Comment:
can we just move those 2 fields inside the main class?

Reply:
I can modify the BLS12381 verify function to accept a G1Point generator param and add it to the KeyPair class